cirq-core/cirq/ops/controlled_operation_test.py

```diff
@@ -47,12 +47,12 @@ def _apply_unitary_(self, args: cirq.ApplyUnitaryArgs) -> Union[np.ndarray, NotI
         zero = seed * a + (0, Ellipsis)
         one = seed * a + (1, Ellipsis)
         result = np.zeros(args.target_tensor.shape, args.target_tensor.dtype)
-        result[zero] = args.target_tensor[zero] * 2 + args.target_tensor[one] * 3
-        result[one] = args.target_tensor[zero] * 5 + args.target_tensor[one] * 7
+        result[zero] = (args.target_tensor[zero] + args.target_tensor[one]) * np.sqrt(0.5)
+        result[one] = (args.target_tensor[zero] - args.target_tensor[one]) * np.sqrt(0.5)
         return result
 
     def _unitary_(self):
-        return np.array([[2, 3], [5, 7]])
+        return np.array([[1, 1], [1, -1]]) * np.sqrt(0.5)
```

Comment:
Changing this to a Hadamard may limit the coverage it provides. I'd prefer if we used a random 1-qubit unitary well outside the "normal" gateset.

Same for its other appearance in this PR.

Reply:
Changed to a random unitary at both places.